_includes/specials/taproot/en/02-descriptors.md

```diff
@@ -0,0 +1,98 @@
+[Output script descriptors][topic descriptors] provide a generic way for
+wallets to store the information needed to create addresses, efficiently
+scan for outputs paying those addresses, and later spend from those
+addresses.  In addition, descriptors are reasonably compact and contain
+a basic checksum, making them convenient for backing up address
+information, copying it between different wallets, or sharing it between
+wallets that are cooperating to provide multiple signatures.
+
+Although descriptors are currently only used by a few projects, they and
+the related [miniscript][topic miniscript] project have the potential to
+significantly improve interoperability between different wallets and
+tools.  This will become increasingly important as more users take
+advantage of the benefits of taproot to improve their security through
+[multisignatures][topic multisignature] and their resilency through
+backup spending conditions.
+
+Before that can happen, descriptors need to be updated to work with
+taproot.  That was the subject of the recently merged [Bitcoin Core #22051][]
+pull request.  The syntax has been designed to allow a single descriptor
+template to provide all the information necessary to use both P2TR
+keypath spends and script path spends.  For simple single-sig, the
+following descriptor is sufficient:
+
+    tr(<key>)
+
+The same syntax may also be used for multisignatures and [threshold
+signatures][topic threshold signature].  For example, Alice, Bob, and
+Carol aggregate their keys using [MuSig][topic musig] and then pay to
+`tr(<combined_key>)`.
+
+Unintuitively, the key specified by `tr(<key>)` won't be the key encoded
+in the address.  The `tr()` descriptor follows a safety recommendation
+from [BIP341][] to use an internal key that commits to an unspendable
```

Comment:
Maybe link directly to https://github.com/bitcoin/bips/blob/master/bip-0341.mediawiki#cite_note-22